content/content-strategy/manage-content-requests/create-content/content-production-template/index.yml

```diff
@@ -0,0 +1,17 @@
+pagetitle: Content production template
+weight: 40
+description: Use the content production template to prepare, write and check for quality content.
+theme: blue
+header:
+  - /_shared/globalheader.md
+  - /_shared/header.md
+main:
+  - intro.md
+  - toc.md
+  - cards.md
+  - using-head.md
+  - using.md
+  - resources-cards-head.md
```

Comment:
Wasn't able to add these comments to the renamed files. 
1. It breaks convention of adding cards to a page _without_ a preceding heading. Consider adding a heading partial between `toc.md` and `cards.md` as done below in this index file.
2. Please update the `toc.md` file to list the new section detailed in the `resources-cards-head.md` file, and the new heading for the first `cards.md` partial. Ta

Reply:
Trisha - noted.